src/Sessions/SessionDisplay.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import {View} from 'react-native';
+
+import {SessionTypes} from '../Types';
+import {SessionList} from './SessionsList';
+import {SessionToggler} from './SessionToggler';
+
+export const SessionDisplay: React.FC<SessionTypes.SessionMessageListProps> = ({
+  sessions,
+}) => {
+  const [currentMessageStatus, setcurrentMessageStatus] = React.useState<
+    SessionTypes.SessionMessageTypeEnum
+  >(SessionTypes.SessionMessageTypeEnum.Scheduled);
+
+  const toggleMessageStatus = (
+    selectedMessageStatus: SessionTypes.SessionMessageTypeEnum
+  ) => {
+    console.log(selectedMessageStatus);
+    setcurrentMessageStatus(selectedMessageStatus);
+  };
+
+  return (
+    <View>
+      <SessionToggler
+        onPress={(
+          selectedMessageStatus: SessionTypes.SessionMessageTypeEnum
+        ) => {
+          return toggleMessageStatus(selectedMessageStatus);
+        }}
+        currentMessageType={currentMessageStatus}
+      />
+      <SessionList
+        sessions={sessions.filter(
+          (m) => m.messageType === currentMessageStatus
```

Comment:
We don't need the messageType value for sessions. Instead, its **status** should determine where it goes. For instance, _Scheduled_ sessions should go under the **Scheduled** tab whereas the others go under the **History** tab.

Reply:
All right. I understand. 
I would update the types and the components.